Sources/Operators.swift

```diff
@@ -241,7 +241,7 @@ public func <~~ (key: String, json: JSON) -> [URL]? {
 /**
 Encode custom operator.
 */
-infix operator ~~> { associativity left precedence 150 }
+infix operator ~~> : MultiplicationPrecedence
```

Comment:
@marcopompei do you know what `MultiplicationPrecedence` signifies? i.e. are there other types and is this the correct one?